dbms/src/Storages/IStorage.h

```diff
@@ -200,6 +200,14 @@ class IStorage : public std::enable_shared_from_this<IStorage>, private boost::n
       */
     virtual void drop() {}
 
+    /** Delete the table data. Called before deleting the directory with the data.
+      * If you do not need any action other than deleting the directory with data, you can leave this method blank.
+      */
+    virtual void truncate(const ASTPtr & /*query*/)
```

Comment:
This comment is a copy-paste of the above. This may lead to confusion.
I've totally rewrite this comment.